api/api/controllers/agent_controller.py

```diff
@@ -1678,3 +1678,33 @@ async def get_agent_summary_os(pretty: bool = False, wait_for_complete: bool = F
     data = raise_if_exc(await dapi.distribute_function())
 
     return json_response(data, pretty=pretty)
+
+
+async def get_agent_uninstall_permission(pretty: bool = False, wait_for_complete: bool = False) -> ConnexionResponse:
+    """Check user's permission to uninstall agents
+
+    Parameters
+    ----------
+    pretty : bool
+        Show results in human-readable format
+    wait_for_complete : bool
+        Disable timeout response
+
+    Returns
+    -------
+    ConnexionResponse
+        API response.
+    """
+    f_kwargs = {}
+
+    dapi = DistributedAPI(f=agent.agent_check_uninstall_permission,
```

Comment:
```suggestion
    dapi = DistributedAPI(f=agent.check_uninstall_permission,
```

Reply:
Fixed in efb4a691b97c462325ee2b7e9c4faa616a435692.